### DISCUSSION TITLE
Pre selected value on server search

### DISCUSSION THREAD

**Discussion by hassanjrao** [sa-si-dev/virtual-select#166] [Q&A] [answered]
Is there any way to set a preselected value on server search? Right now I am not able set the pre-selected value on server search virtual select element.

**Comment by sa-si-dev** [comment]
@hassanjrao as of now, this feature is not available. I will implement it in the future release.

**Reply by hassanjrao** [reply]
Kindly implement it as soon as possible.

**Comment by sa-si-dev** [comment, marked as answer]
@hassanjrao we need list of options to select value, so that we can display the selected option's label.

To prefill value, set `options` (atleast selected option's only) and `selectedValue` properties while initiating the plugin.
